src/main/java/com/es/phoneshop/model/product/ArrayListProductDao.java

```diff
@@ -4,7 +4,7 @@
 import java.util.ArrayList;
```

Comment:
Please use descriptive commit messages in future like:

"Implemented ProductDao"
"Fixed code accorindg to comments"